packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -93,6 +94,17 @@ export async function handleSending(
     deps.telemetry.recordHistogram('ilp_pay_time_ms', payDuration, {
       description: 'Time to complete an ILP payment'
     })
+
+    await deps.telemetry.incrementCounterWithTransactionFeeAmount(
+      'payment_fees',
+      payment.sentAmount,
+      payment.receiveAmount,
+      {
+        description: 'Amount sent through the network as fees',
+        valueType: ValueType.DOUBLE
+      },
+      false // do not preserve privacy
```

Comment:
```suggestion
      false
```
or we just turn it into an object for the options `{ preservePrivacy }`